docs/en/FAQ/Compatible-with-other-javaagent-bytecode-processing.md

```diff
@@ -0,0 +1,52 @@
+## Compatible with other javaagent bytecode processing
+
+### Problem
+1. When use skywalking agent, some other agent, such as Arthas, can't work well 
+https://github.com/apache/skywalking/pull/4858
+
+2. Java agent retransforming class fails with Skywalking agent, such as in this [demo](https://github.com/hengyunabc/skywalking-error-demo)
```

Comment:
Waiting for update at https://github.com/SkyAPMTest/retransform-conflict-demo

Reply:
The demo project is submitted.